platforms/android/AndroidManifest.xml.in

```diff
@@ -93,7 +93,12 @@
                 android:resource="@xml/provider_paths"/>
         </provider>
 
-        <!-- For adding service(s) please check: https://wiki.qt.io/AndroidServices -->
+
+        <service android:name="org.virgil.notification.FirebaseMessageReceiver">
```

Comment:
I would use `org.virgil.messenger` or `@VS_ANDROID_PACKAGE_NAME@` package as a base. So that service name be the next `org.virgil.messenger.notification.FirebaseMessageReceiver`